pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -211,9 +212,9 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (recon
 	}
 
 	// TODO(Raf) Set quotas
-	billingAccount, err := util.GetBillingAccountFromSecret(r.client, operatorNamespace, orgGcpSecretName)
+	billingAccount, err := configmap.GetBillingAccount()
 	if err != nil {
-		reqLogger.Error(err, "could not get org billingAccount from secret", "Secret Name", orgGcpSecretName, "Operator Namespace", operatorNamespace)
+		reqLogger.Error(err, "could not get org billingAccount from configmap", "Configmap", orgGcpConfigMap, "Operator Namespace", operatorNamespace)
```

Comment:
what does the `org` prefix of the variable `orgGcpConfigMap` mean?

Reply:
maybe `orgGCPConfigMap` would be better as `ConfigMapName` or something like that

Reply:
replaced with `ConfigMapName`